vital/types/geodesy.h

```diff
@@ -56,16 +57,52 @@ namespace vital {
  * \see https://en.wikipedia.org/wiki/Spatial_reference_system,
  *      http://www.epsg.org/, https://epsg-registry.org/
  */
-enum class SRID : int
+namespace SRID
 {
-  lat_lon_NAD83 = 4269,
```

Comment:
I assume these numbers are standard enough that we could pass them to `geographiclib` or another package and it would understand them correctly, right?